components/x-follow-button/src/Button.jsx

```diff
@@ -0,0 +1,35 @@
+import { h } from '@financial-times/x-engine';
+import styles from './styles/main.scss';
+
+const Button = ({
+	conceptId,
+	followPlusDigestEmail,
+	isSelected,
+	extraButtonClasses,
+	variant,
+	alternateText,
+	buttonText,
+	name,
+	...props
+}) => (
+	<button
+		data-alternate-text={ isSelected ? `Add ${name} to MyFT` : `Remove ${name} to MyFT` }
+		aria-label={ isSelected ? `Remove ${name} to MyFT` : `Add ${name} to MyFT` }
+		title={ isSelected ? `Remove ${name} to MyFT` : `Add ${name} to MyFT` }
+		data-alternate-label={ isSelected ? `Add ${name} to MyFT` : `Remove ${name} to MyFT` }
+		aria-pressed={ isSelected ? 'true' : 'false' }
+		className={ `${extraButtonClasses ? extraButtonClasses : ''}
+				${styles['n-myft-follow-button']}
+				${variant ? ` n-myft-follow-button--${variant}` : ''}` }
```

Comment:
these classes aren't going to be available in the bundled css output, so you should use ``styles[`n-myft-follow-button--${variant}`]``